src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -150,9 +150,9 @@ public function setIgnoredAttributes(array $ignoredAttributes)
     /**
      * {@inheritdoc}
      */
-    public function hasCacheableSupportsMethod(): bool
+    public function isSupportsVaryByData(): bool
```

Comment:
I'm not english-native, but I feel like this should be either "doesSupportsVaryByDataInterface" or "isSupportsVaryingByDataInterface"
the context should also be taken into account, so that this could be "doesSupportsVaryByDataOrContextInterface"
All in all, I'm not sure the new name is better. But I'll let others decide since I'm the author of the original proposal.

Reply:
`supportsMethodVaries`, `hasVaryingSupportsMethod`? But I'm not really convinced either.

Reply:
I'll improve it. :smile: